fiftyone/operators/registry.py

```diff
@@ -36,8 +40,22 @@ def list_operators(enabled=True):
     Returns:
         a list of :class:`fiftyone.operators.Operator` instances
     """
-    registry = OperatorRegistry(enabled=enabled)
-    return registry.list_operators(include_builtin=enabled != False)
+    global _operator_registry
```

Comment:
There are lots of places where `OperatorRegistry()` is called. If we're going to consider making it a singleton (which folks will hate, I can see it now) then that would need to be done holistically, not just here.

But either way, as willing as I am to introduce singletons, I am hesitant to do that for plugins for fear of losing the ability to dynamically edit a plugin's source and immediately see the results in a running App.

Reply:
Also `OperatorRegistry()` calls `build_plugin_contexts()` which already has a `@plugins_cache` decorator that I believe was added to optimize reloads.

Reply:
> fear of losing the ability to dynamically edit a plugin's source and immediately see the results in a running App.

I think there is a better way of handling this by importing the operator when it's actually being called. It can still be registered beforehand. I don't think we should reimport every operator every time list_operators is called (current behavior).